frontend/src/stimulus/setup.ts

```diff
@@ -52,9 +57,13 @@ OpenProjectStimulusApplication.preregister('pattern-input', PatternInputControll
 OpenProjectStimulusApplication.preregister('scroll-into-view', ScrollIntoViewController);
 OpenProjectStimulusApplication.preregister('ckeditor-focus', CkeditorFocusController);
 OpenProjectStimulusApplication.preregister('auto-submit', AutoSubmit);
+OpenProjectStimulusApplication.preregister('work-packages--activities-tab--index', IndexController);
+OpenProjectStimulusApplication.preregister('work-packages--activities-tab--auto-scrolling', AutoScrollingController);
+OpenProjectStimulusApplication.preregister('work-packages--activities-tab--polling', PollingController);
+OpenProjectStimulusApplication.preregister('work-packages--activities-tab--stems', StemsController);
+OpenProjectStimulusApplication.preregister('work-packages--activities-tab--editor', EditorController);
```

Comment:
I noticed that heavy usage of outlets introduced timing issues which resulted in flakiness in tests- which I reckon is due to operations run on `connect()` across the controllers- where that functionality is dependent on an outlet- E.g. accessing a value from index controller.

Pre-registering addresses this- but I'm not completely satisfied with this solution afaik due to it's impact of the default js bundle.